test/public/envs/detailsPage.test.js

```diff
@@ -92,4 +101,19 @@ module.exports = () => {
         expect(await table[1].evaluate((row) => row.id)).to.equal('row3');
         expect(await table[2].evaluate((row) => row.id)).to.equal('row4');
     });
+
+    it('should successfully display FLP nad ECS links', async () => {
+        await goToPage(page, 'env-details', { queryParameters: { environmentId: 'CmCvjNbg' } });
```

Comment:
No more goToPage except for new test suite. It looks like pptr fails at some point if we navigate too much. If you really need to navigate from the previous page you can use click on the main links (best is to try not to navigate at all)

Reply:
refactored